scripts/nlp_language_modeling/preprocess_music_data_for_megatron.py

```diff
@@ -0,0 +1,266 @@
+# Copyright (c) 2021, NVIDIA CORPORATION.  All rights reserved.
+#
```

Comment:
This file is very similar to the `preprocess_data_for_megatron.py`. what's the difference? maybe reuse the `preprocess_data_for_megatron.py` instead of copying a new one.